src/model.ts

```diff
@@ -279,8 +284,8 @@ export function convertDescribeJobtoJobDetail(
     updateTime: describeJob.update_time,
     startTime: describeJob.start_time,
     endTime: describeJob.end_time,
-    scheduleInterval: 'weekday',
-    downloaded: describeJob.downloaded
+    downloaded: describeJob.downloaded,
+    ...InitialScheduleOptions
```

Comment:
The describe job model should not have any concept of a schedule. It looks like the root cause of this is the fact that `IJobDetailModel` is extending `ICreateJobModel`, which I assume is due to just not wanting to rewrite the types for each property again.

While inheriting the `ICreateJobModel` is shorter, it is fundamentally incorrect as shown here. What's confusing is that although the `DescribeJob` response inherits from the `CreateJob` response (which we are confusingly also calling models), the `IDescribeJobModel` contains the UI state of the `DescribeJob` page, which does not necessarily share all the UI state of the `CreateJob` page. This is part of the model refactor I am proposing here: https://github.com/jupyter-server/jupyter-scheduler/issues/105

To fix this, you should change `IDescribeJobModel` to not inherit from `ICreateJobModel`, and then remove any `ICreateJobModel`-specific properties from this object you are returning here.

Reply:
This is now done, with additional modifications to the advanced options to change expectations for the nature of the `model` prop.